### ISSUE TITLE
any guess why CPS-ASEC worker values look so different?

### ISSUE THREAD

**Issue by ajdamico** [ajdamico/convey#449]
comparing the real world examples for the CPS-ASEC ptotval (total income) and pearnval (worker earnings) among full time full year workers in the united states to the other datasets and the other levels of analysis..

svygini looks reasonable

https://guilhermejacob.github.io/context/1.4-current-population-survey---annual-social-and-economic-supplement-cps-asec.html#worker-earnings

svyarpt looks reasonable

https://guilhermejacob.github.io/context/3.1-at-risk-of-poverty-threshold-svyarpt.html#real-world-examples

but then it looks like something might be going on from svyarpr thru svywatts?

https://guilhermejacob.github.io/context/3.2-at-risk-of-poverty-ratio-svyarpr.html#real-world-examples-1

https://guilhermejacob.github.io/context/3.7-watts-poverty-measure-svywatts-svywattsdec.html#real-world-examples-6

thanks!

**Comment by ajdamico** [comment]
i would say it like this:

For relative poverty measures, it is important to think what should be the reference population for the poverty threshold. For instance, if one is interested is computing poverty rates across regions using a poverty line computed at the contry level (represented by the full sample design object), it is important to run convey_prep immediately after creating the full object. However, imagine that one wants to compute the poverty line using the annual earnings of the "full time full year" working people. In this case, if we subsetted the full object after running convey_prep, the poverty threshold would be computed using the entire sample, including the incomes of partial time partial year workers. In this (non-standard) case, we suggest subsetting the main object before running convey_prep. To use this non-prepped subsetted object to represent the population of reference for poverty lines in its subsets, run convey_prep right after creating it.
In summary, convey_prep sets the population of reference for poverty *threshold* estimation.
